src/bootstrap.php

```diff
@@ -872,6 +873,20 @@ public function load_debug() {
 		$this->singleton->add_class( $class );
 	}
 
+	/**
+	 * Initialise our system status code
+	 *
+	 * @since 5.2.0
+	 *
+	 * @return void
+	 */
+	public function check_system_status() {
+		$class = new Controller\Controller_System_Report( $this->log );
```

Comment:
I'd recommend passing `$this->data->allow_url_fopen` directly to the class on initialisation